src/lib/y2storage/storage_manager.rb

```diff
@@ -516,6 +582,54 @@ def create_logger
         @logger = StorageLogger.new
         ::Storage.logger = @logger
       end
+
+      # Default access mode (read-only or read-write)
+      #
+      # @note During installation, access mode should be read-write.
+      #
+      # @return [Symbol] :ro, :rw
+      def default_storage_mode
+        Yast::Mode.installation || Yast::Mode.autoinst ? :rw : :ro
```

Comment:
This is redundant. If `Mode.autoinst` is true then `Mode.installation` is also true. So you don't need the second part of the condition. Check the table at the Mode documentation (unfortunately not rendered properly in rubydocs).